go.mod

```diff
@@ -1,3 +1,10 @@
 module github.com/lambdaclass/cairo-vm.go
 
 go 1.18
+
+require (
+	github.com/pkg/errors v0.9.1
+	golang.org/x/sys v0.4.0 // indirect
+	golang.org/x/tools v0.5.1-0.20230111220935-a7f7db3f17fc // indirect
+	golang.org/x/tools/cmd/cover v0.1.0-deprecated // indirect
```

Comment:
Are these necessary?

Reply:
Not all of them but I searched if .sum and .mod could be ignored and I found that not. https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file